win_bin/install/install-dcos-cli.ps1

```diff
@@ -95,3 +95,8 @@ dcos config set package.cache $env:temp\dcos\package-cache
 dcos config set package.sources '[\"https://github.com/mesosphere/universe/archive/version-1.x.zip\"]'
 
 dcos package update
+
+echo "Finished installing and configuring DCOS CLI."
+echo ""
+echo "Run this command to set up your environment and to get started:"
+echo "<install_dir>\Scripts\activate.ps1"
```

Comment:
Please give a better message. We know the `install_dir` it should be the env variable `$installation_path`, no?